docs/build.md

```diff
@@ -52,8 +49,8 @@ brew install bazelbuild/tap/bazelisk
 
 ### Configure .bazelrc ###
 Run the ```./configure.sh``` script in the root directory and answer
-"Yes" to the ```manylinux2010``` question when you want to build for
-TensorFlow 2.x or TensorFlow 1.15, and "No" for TensorFlow 1.14 and below.
+"Yes" to the ```manylinux2010``` question if you want to build the
```

Comment:
I guess then we can also remove the yes/no question from the configure script? That will avoid any confusion.

Reply:
as far as I remember, you need the manylinux2010 if you want to build the PIP package in the container but activating by default will cause build issues on macOS.

Reply:
on mac, does not even ask for it. so for mac is removed! 😀 then I guess we can set it to the default for linux builds, @lgeiger ?

Reply:
> I guess then we can also remove the yes/no question from the configure script? That will avoid any confusion.

We still need it if people want to build the converter outside the container.